docs/design/federation-phase-1.md

```diff
@@ -0,0 +1,386 @@
+# Ubernetes Design Spec (phase one)
+
+**Huawei PaaS Team**
```

Comment:
Is this the author of the doc or the intended implementor of the proposal? If they are the author, why not submit it in a separate PR so that they can have author attribution (and respond to comments / questions)?